libs/ui/src/lib/treemap-chart/treemap-chart.component.ts

```diff
@@ -164,10 +169,12 @@ export class GfTreemapChartComponent
         }
       ]
     };
-
     if (this.chartCanvas) {
       if (this.chart) {
         this.chart.data = data;
+        this.chart.options.plugins.tooltip = <unknown>(
```

Comment:
Do you need to set it again when the chart is already there?

Reply:
I assumed it was required for updating the chart since `portfolio-proportion-chart.component.ts` does it this way.